build.gradle

```diff
@@ -295,46 +295,19 @@ shadowJar {
         resource = 'reference.conf'
     }
 
-    // This has been added to ensure that Spark will produce info messages (by default)
-    exclude 'log4j.properties'
-
-    doLast {
-        // Create a symlink to the newly created jar.  The name will be hellbender-protected.jar and
-        //  it will be at the same level as the newly created jar.  (overwriting symlink, if it exists)
-        // Please note that this will cause failures in Windows, which does not support symlinks.
-        createAllSymlinks(destinationDir.toString(), archivePath, "")
-    }
-}
 
-tasks.withType(ShadowJar) {
-    manifest {
-        attributes 'Implementation-Title': 'Hellbender',
-                'Implementation-Version': version,
-                'Main-Class': 'org.broadinstitute.hellbender.Main'
-    }
-    from(project.sourceSets.main.output)
-    baseName = project.name + '-all'
-    mergeServiceFiles()
     relocate 'com.google.common', 'org.broadinstitute.hellbender.relocated.com.google.common'
     zip64 true
     exclude 'log4j.properties' // from adam jar as it clashes with hellbender's log4j2.xml
 }
 
-configurations {
-    sparkConfiguration {
-        extendsFrom runtime
-        // exclude Hadoop and Spark dependencies, since they are provided when running with Spark
-        // (ref: http://unethicalblogger.com/2015/07/15/gradle-goodness-excluding-depends-from-shadow.html)
-        exclude group: 'org.apache.hadoop'
-        exclude module: 'spark-core_2.10'
-        exclude group: 'org.slf4j'
-        exclude module: 'jul-to-slf4j'
-        exclude module: 'javax.servlet'
-        exclude module: 'servlet-api'
-        exclude group: 'com.esotericsoftware.kryo'
-        exclude module: 'spark-mllib_2.10'
-        exclude group: 'org.scala-lang'
-        exclude module: 'kryo'
+shadowJar {
+    classifier = 'spark_standalone'
```

Comment:
shadowJar and sparkJar have different classifiers, as far as I know they're totally arbitrary and used to make it more human readable output